components/browser/state/src/main/java/mozilla/components/browser/state/reducer/LastAccessReducer.kt

```diff
@@ -17,9 +17,8 @@ internal object LastAccessReducer {
      */
     fun reduce(state: BrowserState, action: LastAccessAction): BrowserState = when (action) {
         is UpdateLastAccessAction -> {
-            state.updateTabOrCustomTabState(action.tabId) { sessionState ->
-                val tabSessionState = sessionState as TabSessionState
-                tabSessionState.copy(lastAccess = action.lastAccess)
+            state.updateTabState(action.tabId) { sessionState ->
+                sessionState.copy(lastAccess = action.lastAccess)
```

Comment:
This is just cleanup as it would've never worked for custom tabs (see the cast to `TabSessionState` which would've crashed.) We only need this for regular tabs anyway.